### DISCUSSION TITLE
Splitting track to exclude denning season data while preserving individuals

### DISCUSSION THREAD

**Discussion by nad22gn** [bmcclintock/momentuHMM#189] [Q&A]
I am using black bear movement data and am trying to exclude the denning season from my analysis. I want to avoid connecting the pre-denning track to the post-denning track because those would be invalid steps. However, I can't find a way to do this that doesn't also force the separated tracks to be considered different individuals. Is there a way to do this that preserves individuals? Or would I have to keep the denning season in the analysis and add another behavioral state to attempt to capture denning?

**Comment by bmcclintock** [comment]
It's not clear to me why you need to preserve individuals -- are you wanting to include individual-level covariate effects? If so, you can separate the tracks while adding an additional individual variable that preserves individuals (named something other than `ID`, e.g., `individual`) for inclusion as a covariate (e.g., `formula = ~ individual`). Alternatively, you could potentially include denning as an additional state (and use the `knownStates` argument in `fitHMM` to indicate the denning period, if known). The latter approach could be useful if you're interested in learning about the timing and duration of denning.
